frontend/src/components/Pages/JoinVrikshit/joinVrikshit.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './joinVrikshit.css';
+
+class JoinVrikshit extends React.Component {
+  componentDidMount() {
+    window.scrollTo(0, 0);
+  }
+  render() {
+    return (
+      <div className="container description_Form">
+        <div className="row">
+          <div className="col-md-12">
+            <h1 className="volunteer_headline">VOLUNTEER WITH US</h1>
+            <iframe
+              src="https://docs.google.com/forms/d/e/1FAIpQLScPBGeteil_ggg3pHZaOou7KNEMxf04WgbPKoV43WksmJFzaQ/viewform?embedded=true"
+              width="1200"
+              height="675"
+              frameborder="0"
```

Comment:
this width/height will make page unresponsive.

Reply:
Right , removing the width and height

Reply:
@itsnamangoyal  - From next time , please cut different branch name it **feature-css**

Reply:
alright